ui/packages/catalog-ui-search/src/main/webapp/js/cql.js

```diff
@@ -502,7 +503,9 @@ function write(filter) {
             ymax = filter.value[3]
           return (
             'BBOX(' +
+            '"' +
```

Comment:
✏️ Consider replacing concatenation with string interpolation in this file to make things more readable

Reply:
I thought about doing that but I was able to make sense of things well enough. And to be frank that would make this a much bigger change because I'd be updating all instances of concatenation and there's a lot happening.